assets/src/edit-story/components/form/label.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+const Label = styled.span`
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 11px;
+	line-height: 16px;
+	flex-basis: 80px;
```

Comment:
Replying to https://github.com/ampproject/amp-wp/pull/4138#discussion_r372119701:

`width` for a span is respected when it's part of a flexbox row. But setting `flex-basis` makes a lot more sense, as that's what it essentially is.

Reply:
Yeah. You're essentially doing an implicit conversion of `span` to `div`. Why not just use `div`?

Reply:
I felt that `span` is slightly more semantic. Have no strong feelings about it though.